src/encoded/search.py

```diff
@@ -1322,13 +1323,31 @@ def set_facets(search, facets, search_filters, string_query, request, doc_types,
         else:
 
             facet['aggregation_type'] = 'terms'
-            term_aggregation = {
-                "terms" : {
-                    'size'    : 100,            # Maximum terms returned (default=10); see https://github.com/10up/ElasticPress/wiki/Working-with-Aggregations
-                    'field'   : query_field,
-                    'missing' : facet.get("missing_value_replacement", "No value")
+            
+            term_aggregation = None
```

Comment:
I don't think you need this declaration since `term_aggregation` is set unconditionally in the if/else below